docs/02_types/interfaces.md

```diff
@@ -5,4 +5,194 @@ path: /docs/types/interfaces
 
 # Interfaces
 
-Documentation coming soon
+Interfaces are an abstract type which may be implemented by object types.
+
+An interface has fields, but it’s never instantiated. Instead, objects may
+implement interfaces, which makes them a member of that interface. Also, fields
+may return interface types. When this happens, the returned object may be any
+member of that interface.
+
+For example, let's say a `Customer` (interface) can either be an `Individual`
+(object) or a `Company` (object). Here's what that might look like in the
+[GraphQL Schema Definition Language](https://graphql.org/learn/schema/#type-language) (SDL):
+
+```graphql
+interface Customer {
+  name: String!
+}
+
+type Company implements Customer {
+  employees: [Individual!]!
+  name: String!
+}
+
+type Individual implements Customer {
+  employed_by: Company
+  name: String!
+}
+
+type Query {
+  customers: [Customer!]!
+}
+```
+
+Notice that the `Customer` interface requires the `name: String!` field. Both
+`Company` and `Individual` implements that field so that they can implement the
```

Comment:
`implements` -> `implement`

This sentence is kinda strange though. I wonder if something else could be said so we're not saying "_implement_ x so they can _implement_ y"

Reply:
Reworded the sentence to avoid the double `implement`